src/odemis/acq/millmng.py

```diff
@@ -0,0 +1,519 @@
+            return 0
+
+        nb_anchor_scanning = math.ceil(setting.duration.value / setting.dcPeriod.value)
+        drift_estimation = AnchoredEstimator(self._scanner, self._detector,
+                                             setting.dcRoi.value, setting.dcDwellTime.value, ANCHOR_MAX_PIXELS,
+                                             follow_drift=False)
+        return drift_estimation.estimateAcquisitionTime() * nb_anchor_scanning
+
+    def _move_to_site(self, site: CryoFeature):
+        """
+        Move the stage to the given site.
+        :param site: (CryoFeature) The site to move to.
+        """
+        target_pos = {"x": site.pos.value[0],
+                      "y": site.pos.value[1],
+                      "z": site.pos.value[2]}
+        # Todo check if it is wise to move in Z direction without affecting the OPTICAL-FIB alignment
+        logging.debug("For feature %s moving the stage to %s m", site.name.value, target_pos)
+        self._future.running_subf = self._stage.moveAbs(target_pos)
+        stage_time = self.estimate_stage_movement_time(site)
+        t = stage_time + 5  # adding extra margin
+        try:
+            self._future.running_subf.result(t)  # or 2 min should be fast enough to move anywhere
+        except TimeoutError:
+            logging.warning("Failed to move the stage for feature %s within %s s", site.name.value, t)
+            self._future.running_subf.cancel()
+            # TODO move.cryoSwitchSamplePosition(MILLING)
+
+    def _mill_all_settings(self, site: CryoFeature):
+        """
+        Iterates over all the milling settings for one site
+        """
+        # Move the stage
+        self._move_to_site(site)
+        # if exception is timeout error, continue to the next site
+        if Exception == TimeoutError:
+            return
+
+        for setting_nb, milling_settings in enumerate(self._settings):
+            # Tilt stage angle
+            stage_rx = self._ion_beam_angle - milling_settings.beamAngle.value
+            logging.debug("Tilting stage to %f °", math.degrees(stage_rx))
+            self._future.running_subf = self._stage.moveAbs({"rx": stage_rx})
+
+            try:
+                t = 15  # s
+                self._future.running_subf.result(timeout=t)
+            except TimeoutError:
+                logging.debug("Failed to tilt the stage for feature %s within %s s", site.name.value, t)
+                raise
+            # Configure HoV
+            self._scanner.horizontalFoV.value = milling_settings.horizontalFoV.value
+
+            # Make the blanker automatic (ie, disabled when acquiring)
+            self._scanner.blanker.value = None
+
+            # set the milling current
+            self._scanner.probeCurrent.value = milling_settings.current.value
+
+            # Initialize the drift corrector only if it is requested by the user
+            # max_pixels can change according to drift correction approach
```

Comment:
I don't think that this `max_pixels` comment is very useful. For now it never changes. So let's drop this comment, or mark a TODO with an explanation what would need to be done.